src/popover/popover.ts

```diff
@@ -185,9 +186,7 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
 
     this._zoneSubscription = _ngZone.onStable.subscribe(() => {
       if (this._windowRef) {
```

Comment:
I think it might be safer to have `if (this._windowRef)` inside the `this.positionElements()` function.
Now you're checking it only here, but not before the other call to `positionElements()`